packages/workflows/src/definition/create-products.ts

```diff
@@ -275,10 +275,16 @@ const handlers = new Map([
             },
             {
               from: Actions.createProducts,
-              alias: ProductHandlers.listProducts.aliases.products,
+              alias: "products",
```

Comment:
q: whats the reasoning behind removing the alias?

Reply:
I see you removed the handler.

Reply:
exactly, I only need it here for the middleware